cmd/snapweb/main.go

```diff
@@ -34,7 +35,7 @@ func init() {
 }
 
 func main() {
-	initURLHandlers(logger)
+	initURLHandlers(snappy.NewDefaultNTPConfigurationFilesLocator(), logger)
```

Comment:
Maybe this is a classic case of YAGNI on my side - but it feels a bit strange to init the handlers (all handlers) with a ntpconfigrationsfileslocatior. Once the handlers grow a new thing that gets configured we will need a new FooBarConfigurationFilesLocator and that grows out of control quickly. Maybe one of the following: a) have a `type SnapWebConfig struct { NtpConfigFilePath string }` and we just add a new FooBarFilePath when we need it. Or (even simpler) just having a "var ntpConfigFilePath = "/etc/foo.conf" in the code and in the test we just replace that with a mocked file location. This requires even less code and is a common pattern in the go world.